File: cs50/week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
int n = 0;
// loop to take input from 1 to 8
do
{
    n = get_int ("Enter height of pyramid: ");
}while (n < 1 || n > 8);

for (int i = 1; i <= n; i++)
{
    for (int j = 1; j <= n+2+i; j++)
    {
        if (j <= n-i)
        {
            printf (" ");
        }
        else if (j > n-i && j < n+1)
        {
            printf ("#");
        }
        else if (j > n+2)
        {
            printf ("#");
        }
        else
        {
            printf (" ");
        }
    }
    printf("\n");
}

}

File: cs50/week6/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = 0
    while height <= 0 or height >= 9:
        height = get_int("Enter height: ")
    pyramid(height)


def pyramid(height):
    for i in range(height):
        for j in range(height - (i + 1)):
            print(" ", end="")
        for j in range(i + 1):
            print("#", end="")
        print("")


main()
